client/index.html

```diff
@@ -3,125 +3,126 @@
 <!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
 <!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
 <!--[if gt IE 8]><!--> <html class="no-js"> <!--<![endif]-->
-  <head>
```

Comment:
Please specify the section to which you are referring. I cannot find anything about making a file completely flat.